spec/tapioca/compilers/dsl/action_controller_helpers_spec.rb

```diff
@@ -6,6 +6,12 @@
 
 RSpec.describe(Tapioca::Compilers::Dsl::ActionControllerHelpers) do
   describe("#initialize") do
+    def rbi_for(content)
```

Comment:
`rbi_for`? maybe `constants_from`?

Reply:
copy past development tricked me

Reply:
😂

Reply:
obviously the same rename applies to all instances, i just didn't comment on each.